.github/CONTRIBUTING.md

```diff
@@ -167,50 +167,56 @@ Additional references:
-* Additional feedback is always welcome!
-  Even if you are not designated as a Reviewer,
-  feel free to review others' pull requests anyway.
-  Leave your comments even if everything looks good;
-  a simple "Looks good to me" or "LGTM" will suffice.
-  This way we know someone has already taken a look at it!
-* When updating your pull request, please **create new commits**
-  and **don't rewrite the commits history**. This way it's very easy for
-  the reviewers to see diff between iterations.
-  If you rewrite the history in the pull request, review could be much slower.
-  Once the review is done, you can rewrite the history to make it prettier,
-  if you like.
-  Otherwise it's likely would be squashed on merge to master.
-* Once the code review is done,
-  all merge conflicts are resolved,
-  and the CI system build status is passing,
-  the PR Assignee will merge your changes.
-* For more information on the PowerShell Maintainers' process,
-  see the [documentation](../docs/maintainers).
+#### Pull Request - Workflow
+
+1. The PR *author* creates a pull request from a fork.
+1. The *author* ensures that their pull request passes the [CI system][ci-system] build.
+   - If the build fails, a [Repository Maintainer][repository-maintainer] adds the `Review - waiting on author` label to the pull request. 
+   The *author* can then continue to update the pull request until the build passes.
+1. If the *author* knows whom should participate in the review, they should add them otherwise they can add the recommended *reviewers*.
+1. Once the build passes, if there is not sufficient review, the *maintainer* adds the `Review - needed` label.
+1. An [Area Expert][area-expert] should also review the pull request.
+   - If the *author* does not meet the *reviewer*'s standards, the *reviewer* makes comments. A *maintainer* then removes the `Review - needed` label and adds 
+   the `Review - waiting on author` label. The *author* must address the comments and repeat from step 2.
+   - If the *author* meets the *reviewer*'s standards, the *reviewer* approves the PR. A maintainer then removes the `need review` label.
+1. Once the code review is completed, a *maintainer* merges the pull request after one business day to allow for additional critical feedback.
+
+#### Pull Request - Roles and Responsibilities
+
+1. The PR *author* is responsible for moving the PR forward to get it Approved.
+   This includes addressing feedback within a timely period and indicating feedback has been addressed by adding a comment and mentioning the specific *reviewers*.
+   When updating your pull request, please **create new commits** and **don't rewrite the commits history**. 
+   This way it's very easy for the reviewers to see diff between iterations.
+   If you rewrite the history in the pull request, review could be much slower.
+   Once the review is done, you can rewrite the history to make it prettier, if you like.
```

Comment:
Let's not ask authors to rewrite history after approval. The author might add codes that was not there during the review, and it would be impossible for us to know.

Reply:
I'll remove this, it was there previously, but I agree it doesn't make sense